src/libsync/abstractnetworkjob.cpp

```diff
@@ -463,13 +463,89 @@ QString networkReplyErrorString(const QNetworkReply &reply)
+            userFriendlyMessage = QObject::tr("The server couldn’t process your request because some information was incorrect or incomplete. Please try syncing again later, or contact your server administrator for assistance.");
+            break;
+        case 423: //Locked
+            userFriendlyMessage = QObject::tr("The resource you are trying to access is currently locked and cannot be modified. Please try changing it later, or contact your server administrator for assistance.");
+            break;
+        case 428: //Precondition Required
+            userFriendlyMessage = QObject::tr("This request could not be completed because it is missing some required conditions. Please try again later, or contact your server administrator for help.");
+            break;
+        case 429: //Too Many Requests
+            userFriendlyMessage = QObject::tr("You made too many requests. Please wait and try again. If you keep seeing this, your server administrator can help.");
+            break;
+        case 500: //Internal Server Error
+            userFriendlyMessage = QObject::tr("Something went wrong on the server. Please try syncing again later, or contact your server administrator if the issue persists.");
+            break;
+        case 501: //Not Implemented
+            userFriendlyMessage = QObject::tr("The server does not recognize the request method. Please contact your server administrator for help.");
+            break;
+        case 502: //Bad Gateway
+            userFriendlyMessage = QObject::tr("We’re having trouble connecting to the server. Please try again soon. If the issue persists, your server administrator can help you.");
+            break;
+        case 503: //Service Unavailable
+            userFriendlyMessage = QObject::tr("The server is busy right now. Please try syncing again in a few minutes or contact your server administrator if it’s urgent.");
+            break;
+        case 504: //Gateway Timeout
+            userFriendlyMessage = QObject::tr("It’s taking too long to connect to the server. Please try again later. If you need help, contact your server administrator.");
+            break;
+        case 505: //HTTP Version Not Supported
+            userFriendlyMessage = QObject::tr("The server does not support the version of the connection being used. Contact your server administrator for helps.");
+            break;
+        case 507: //Insufficient Storage
+            userFriendlyMessage = QObject::tr("The server does not have enough space to complete your request. Please check how much quota your user has by contacting your server administrator.");
+            break;
+        case 511: //Network Authentication Required
+            userFriendlyMessage = QObject::tr("Your network needs extra authentication. Please check your connection. Contact your server administrator for help if the issue persists.");
+            break;
+        case 513: //resource not authorized
```

Comment:
```suggestion
        case 513: // Resource Not Authorized
```

Unless this was on purpose

Reply:
> it is just a comment

I am aware, but I like keeping things tidy :)